### ISSUE TITLE
missing oodebugrun-swipl in Docker

### ISSUE THREAD

**Issue by janbbeck** [cmu-sei/pharos#121]
https://github.com/cmu-sei/pharos/tree/master/share/prolog/oorules
Here:
```https://github.com/cmu-sei/pharos/tree/master/share/prolog/oorules``` 
it is stated that there is a script,
```./share/prolog/oorules/oodebugrun-swipl ooprog-facts.pl >ooprog.log```

which I cannot find in the docker image. I am fairly certain it was there a few weeks back, albeit at a different location. Have things changed in the mean time and perhaps the script is deprecated? Or is it there and I am just not looking right?

I deleted all my Docker images and pulled :latest a few minutes ago.

**Comment by sei-eschwartz** [comment]
Try oodebugrun instead.  We removed XSB support so all the scripts are for swi now and don't have the suffix.  We should update the docs to reflect that if we haven't.

Sent from Workspace ONE Boxer

On Jul 11, 2020 3:57 PM, Jan Beck <notifications@github.com> wrote:

https://github.com/cmu-sei/pharos/tree/master/share/prolog/oorules
Here:
https://github.com/cmu-sei/pharos/tree/master/share/prolog/oorules
it is stated that there is a script,
./share/prolog/oorules/oodebugrun-swipl ooprog-facts.pl >ooprog.log

which I cannot find in the docker image. I am fairly certain it was there a few weeks back, albeit at a different location. Have things changed in the mean time and perhaps the script is deprecated? Or is it there and I am just not looking right?

I deleted all my Docker images and pulled :latest a few minutes ago.
